xknx/core/state_updater.py

```diff
@@ -2,13 +2,16 @@
 import asyncio
 from enum import Enum
 from functools import partial
+import logging
 
 from xknx.exceptions import ConversionError
 from xknx.remote_value import RemoteValue
 from xknx.telegram import GroupAddress
 
 DEFAULT_UPDATE_INTERVAL = 60
 
+logger = logging.getLogger("xknx_log")
```

Comment:
This should have an extra logger imho - it spams a lot.

Reply:
This wouldn't change anything. It would still get logged, always. In the logfile and in the usual stream logs.

Reply:
We can then set individual log levels for each logger.